### PR TITLE
Clarify that issue triaging requires labeling

### PR DESCRIPTION
**Pull request by tigrannajaryan**
We discussed this in the spec SIG meeting. We decided that we want
to make the first response to the new issues clearer. To enable that
I added 3 labels to the spec repo to use for the first response.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -170,13 +170,17 @@ The following diagram shows the initial triaging of new issues.
...
+    before we can make a decision. More information may be requested from the
+    issue author or others.
   - No, this issue should not be accepted. This can happen when the issue is a
     duplicate, was discussed and rejected in the past, is out of the project
     scope, etc. The assignee labels the issue "won't do", closes it and provides
     the reason for closing. The issue author has the choice to appeal the
     decision, in which case the issue is reopened and the assignee asks other
     spec approvers to help triage and facilitate the discussion.
+  - As a result of triaging one of the 3 labels must be applied to the issue:
+    `triaged-accepted`, `triaged-needmoreinfo`, `triaged-rejected`. Absence
```
Comment (reyang) [inline CONTRIBUTING.md:182]: If the issue is marked as `triaged-rejected`, does it mean the issue will be closed?

**Review by reyang — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (tigrannajaryan) [reply CONTRIBUTING.md:182]: Yes, if the answer is "No" the issue is closed. It is described above.

**Comment by tigrannajaryan** [comment]
@open-telemetry/specs-approvers please review.

**Comment by tigrannajaryan** [comment]
@open-telemetry/specs-trace-approvers @open-telemetry/specs-metrics-approvers @open-telemetry/log-collection-approvers can we get a couple more reviews/approvals on this?

**Review by MrAlias — APPROVED** [review]

**Review by cijothomas — APPROVED** [review]

**Pull request merged**